### ISSUE TITLE
Manjaro: GUI starts, daemon doesn't (with a quirk)

### ISSUE THREAD

**Issue by 6-AND-9** [evilsocket/opensnitch#515]
Hi there,

I am a Linux newbie, so please be gentle. I previously had OpenSnitch on my Ubuntu installation, and it worked great. 

However, I recently switched to Manjaro Pahvo (Gnome) and tried to install/build it from AUR, by using the software centre app. Installation went fine and I can find and run it from the menu. Unfortunately, even though the GUI starts it doesn't work. Clicking on stuff does nothing essentially. It also doesn't pull up the dialog at access requests, like it did on Ubuntu.

I also tried starting it from console. `opensnitch` command didn't work, but `opensnitchd` did. However, this was the output:

```
[2021-09-24 21:28:51]  IMP  Starting opensnitch-daemon v1.4.0
[2021-09-24 21:28:51]  INF  Loading rules from /home/[username]/rules ...
[2021-09-24 21:28:51]  !!!  Path '/home/[username]/rules' does not exist
```

Running  `/usr/bin/opensnitchd -rules-path /etc/opensnitchd/rules` resulted in error: 
```
[2021-09-24 21:33:35]  IMP  Starting opensnitch-daemon v1.4.0
[2021-09-24 21:33:35]  INF  Loading rules from /etc/opensnitchd/rules ...
[2021-09-24 21:33:35]  WAR  Is opensnitchd already running?
[2021-09-24 21:33:35]  !!!  Error while creating queue #0: Error unbinding existing q handler from AF_INET protocol family: operation not permitted
```

but running it with `sudo` gave me this:
```
[2021-09-24 21:33:48]  IMP  Starting opensnitch-daemon v1.4.0
[2021-09-24 21:33:48]  INF  Loading rules from /etc/opensnitchd/rules ...
OK: libnetfiler_queue supports nfq_get_uid
OK: libnetfiler_queue supports nfq_get_uid
```

Basically, the daemon started working now. And now it started intercepting stuff.

What's going on here, and how can I ensure it will work in the future?

**Comment by gustavo-iniguez-goya** [comment]
Hi @6-AND-9 !

I think you need to enable the service: `systemctl enable opensnitch; systemctl start opensnitch`

The deb and rpm packages does it for you, but with the AUR buildpkg you have to do it manually.

**Comment by 6-AND-9** [comment]
Hi @gustavo-iniguez-goya,

And thank you very much for your reply! You were right, this solved the issue. Thanks again.

Now I have another issue, but I will open a separate thread for that.

**Comment by glmsmoura** [comment]
Hello! I am having the same issue stated by @6-AND-9 , but the service can't be found. I use arch, and downloaded it from the AUR.

**Comment by glmsmoura** [comment]
Uh, I tried `systemctl start opensnitchd` and it worked!
